### DISCUSSION TITLE
Survey About AI-Assisted Code Generation Tools

### DISCUSSION THREAD

**Discussion by kim-jeong-hyeon** [community/community#54866] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

We are a research group under the direction of Prof. [Doowon Kim](https://doowon.github.io/) at the University of Tennessee, Knoxville, in collaboration with Sungkyunkwan University, South Korea. We are looking to recruit participants with some experience in software development.

This survey is intended to better understand developers' perceptions of AI-assisted code generation tools (e.g., IntelliSense, GitHub Copilot, ChatGPT, etc.).

The survey asks a series of questions about your programming experience and general demographics.

The survey consists of three parts:

Part 1: Demographic questions

Part 2: Simple programming quiz

Part 3: Programming experience with AI-assisted code generation tools

You are eligible for this study if you:

Are at least 18 years old,

Have programming experience,

Are comfortable completing this study in English.

<Edited by Staff>

We look forward to hearing your thoughts on code-generation tools!

Thank you.

**Comment by Akash1134** [comment, marked as answer]
Hi @kim-jeong-hyeon Thanks for sharing your survey with the community. Unfortunately, we currently do not allow self-promotion, surveying or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
